### ISSUE TITLE
Support WMS for aerial imagery

### ISSUE THREAD

**Issue by vfosnar** [Helium314/SCEE#525]
**Use case**
Czechia CUZK orthophoto (from the iD editor) is much better than Esri here in the Czech Republic, but it uses the WMS instead of a standard tile server. (bbox instead of x,y,z)

**Proposed Solution**
Add option to specify {bbox} instead of {x}, {y}, and {z}

**Comment by Helium314** [comment]
I don't think Tangram can do this.
It should be possible in MapLibre, but there WMS does not work offline.

**Comment by mnalis** [comment]
yes, WMS is significantly more complex (even if iD hides that from user)...

@vfosnar possible workaround (if Czech community has their own server somewhere) is to install [mapproxy](https://www.mapproxy.org/), which is able to on-demand convert from WMS to TMS (among other things like changing projections, caching, etc).  

([mapproxy-hr](https://github.com/osm-hr/mapproxy-hr/) has example config files for WMS->TMS conversion for Croatian aerials, if you're interesting in taking that route.)

**Comment by Helium314** [comment]
SCEE is now using MapLibre, if anyone wants to give it a try...

**Comment by vfosnar** [comment]
For now I'm hosting a simple service for redirecting XYZ tile requests to čúzk imagery wms to bypass this: https://codeberg.org/vfosnar/scee-bridge

But it's running on my homelab-ish oracle server so expect unexpected downtime cuz of broken nginx config.

**Comment by andrewharvey** [comment]
I tried https://github.com/Helium314/SCEE/compare/modified...andrewharvey:bbox-imagery?expand=1 but when I open in Android Studio, it builds successfully, but immediately crashes on my device, so I can't test it.

**Comment by mcliquid** [comment]
> I tried https://github.com/Helium314/SCEE/compare/modified...andrewharvey:bbox-imagery?expand=1 but when I open in Android Studio, it builds successfully, but immediately crashes on my device, so I can't test it.

I tried something similar and it actually works for me. I tried three different aerial image providers, and it works.
You can try the VoGIS 2023 aerials in Vorarlberg, Austria with:
`https://vogis.cnv.at/mapserver/mapserv?map=i_luftbilder_r_wms.map&SERVICE=WMS&VERSION=1.3.0&REQUEST=GetMap&LAYERS=ef2023_10cm_t&STYLES=&FORMAT=image/png&TRANSPARENT=TRUE&CRS=EPSG:3857&BBOX={bbox-epsg-3857}&WIDTH=256&HEIGHT=256`
Screenshot:
<img width="100" alt="Image" src="https://github.com/user-attachments/assets/2fa7e66d-93af-4dfa-bc1d-5ccc2ee4312d" />

I tried two more, and it works.

Of course, this is not a complete WMS layer, but it helps me enormously. Should I create a PR?

**Comment by Helium314** [comment]
> Should I create a PR?

Sure!
